drivers/pwm/pwm_mcux_ftm.c

```diff
@@ -174,7 +174,7 @@ static int mcux_ftm_pin_configure_capture(const struct device *dev,
 	data->capture[pair].user_data = user_data;
 	param = &data->capture[pair].param;
 
-	if (flags & PWM_CAPTURE_MODE_CONTINUOUS) {
+	if ((flags & PWM_CAPTURE_MODE_MASK) == PWM_CAPTURE_MODE_CONTINUOUS) {
```

Comment:
Looks like this change is in the wrong commit `tests: drivers: pwm: add PWM loopback test`

Reply:
Fixed.